packages/gasket-plugin-logger/lib/index.js

```diff
@@ -8,12 +8,14 @@ function createChildLogger(parent, metadata) {
     error: (...args) => console.error(...args, metadata),
     info: (...args) => console.info(...args, metadata),
     warn: (...args) => console.warn(...args, metadata),
+    fatal: (...args) => console.error(...args, metadata),
+    trace: (...args) => console.trace(...args, metadata),
```

Comment:
I believe we landed on only providing `console` log method/levels in the core logger plugin. That way we don't have to chase other implementation possibilities.

How are these other levels used by fastify? If the extra methods are required, could you perhaps create a child logger for it?

```
const logger = gasket.logger.child({
  fatal: gasket.logger.fatal ?? gasket.logger.error,
  trace: gasket.logger.trace ?? gasket.logger.debug
})

somethingThatNeedsMoreLevels(logger)
```